packages/swap/src/event-handlers/swapDepositAddressReady.ts

```diff
@@ -15,6 +15,7 @@ const swapDepositAddressReadyArgs = z.object({
   brokerCommissionRate: z.number().int(),
   sourceChainExpiryBlock: u128.optional(),
   channelMetadata: ccmMetadataSchema.optional(),
+  boostFee: z.number().int(),
```

Comment:
```suggestion
  boostFee: z.number().int().optional(),
```

Reply:
Its actually always going to be present inside the event payload.
<img width="249" alt="Screenshot 2024-02-13 at 15 01 28" src="https://github.com/chainflip-io/chainflip-sdk-monorepo/assets/18466333/96206ff7-a00d-4470-bf99-bca3232a3ee0">